src/components/dashboard/sidebar/SubsidiesSummary.jsx

```diff
@@ -77,15 +76,15 @@ const SubsidiesSummary = ({
 
   return (
     <>
-      {hasAssignedSubsidyOrRequests && (
+      {hasAssignedSubscriptionOrCodeOrOffer && (
         // TODO: Design debt, don't have cards in a card
         <SidebarCard
           cardSectionClassNames={
-            `${programProgressPage && 'border-remove'}`
+            `border-remove ${programProgressPage}`
```

Comment:
Previously, this logic indicated that the `border-remove` class would only be applied when `programProgressPage` is true. With your change, I think we'd get a boolean `true`/`false` class name (e.g., `class="border-remove false"`). If we always want `border-remove` here now, is `programProgressPage` needed here anymore?